apps/prenumerera/src/Prenumerera/PaymentSelect.purs

```diff
@@ -93,10 +93,10 @@ didMount self@{ props: { location: { state: Just state } } } = do
       maybe (pure unit) (send self) maybeAction
 didMount _ = pure unit
 
-didUpdate :: Self -> Effect Unit
-didUpdate self = do
-  Console.log "did update payment select"
-  Console.log $ unsafeCoerce self.state
+didUpdate :: Self -> { prevProps :: Props, prevState :: State } -> Effect Unit
```

Comment:
I guess this is an unrelated change that slipped in?

Reply:
We should actually fix this in a separate PR.